Frontend/src/GlobalStyles.css

```diff
@@ -1,3 +1,14 @@
+:root{
+    --black-20: rgb(20, 20, 20);
+    --gray-52: rgb(52, 52, 52);
+    --gray-72: rgb(72, 72, 72);
+    --gray-203: rgb(203, 213, 225);
+    --white-248: rgb(248,250,252);
+    --white-255: rgb(255, 255, 255);
+    --skyblue-190: hsl(190, 97%, 47%);
+    --darkblue-245: hsl(245, 95%, 22%);
+    --blue-208: hsl(208, 97%, 58%);
+}
```

Comment:
Please use/add the color variables from the Figma [design](https://www.figma.com/file/mkhBuJAS2arzy3wrbTZp2D/Simple-Landing-page-for-OSCSA%2C-with-Log-In-and-Sign-Up-Pages?node-id=0%3A1)

Reply:
@sandygudie Looking at the Figma file I added all the color variables but if there are any you feel I missed please let me know, specifically in the section I missed.